### DISCUSSION TITLE
using gsca with users

### DISCUSSION THREAD

**Discussion by sjay8** [kuzmoyev/google-calendar-simple-api#201] [Q&A] [answered]
im creating a small project that i need some users to interface with my app. can i use gsca with users? since i won't have their GCP credentials?  thanks!

**Comment by kuzmoyev** [comment, marked as answer]
GCP credentials are for your app, and you'll authenticate users with [oauth2](https://developers.google.com/identity/protocols/oauth2), receive their access token, and use it in your requests. (Assuming you want to access their calendars).

**Reply by sjay8** [reply]
perfect thanks!
